introduction.md

```diff
@@ -1,2 +1,5 @@
 Hi.  My name is Dave.
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?